core/server/pom.xml

```diff
@@ -249,6 +249,20 @@
           <!-- insert other configuration here (optional) -->
         </configuration>
       </plugin>
+      <plugin>
```

Comment:
This change has been added to the root pom file, so we don't need to modify `core/server/pom.xml` any more